release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -1045,67 +1349,377 @@ The current release fixes this issue. Now, when a rollover occurs in the `indexm
+
+* Previously, running the `operator-sdk cleanup` command did not clean up Operators that were deployed with the `operator-sdk run bundle` command. Instead, an error message was displayed and the Operator was not cleaned up. Now, the `operator-sdk cleanup` command has been updated, and Operators deployed with `run bundle` can be cleaned up by using the `cleanup` command. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1883422[*BZ#1883422*])
 
 *Performance Addon Operator*
 
-* Previously, incorrect wait in the must-gather logic resulted in too early termination of log gathering. This issue resulted in depending on timing the log gathering operation being interrupted prematurely, leading to partial log collection.
-This is now fixed by adding the correct wait in the must-gather logic. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1906355[*BZ#1906355*])
+* Previously, incorrect wait in the must-gather logic resulted in too early termination of log gathering. This issue resulted in depending on timing the log gathering operation being interrupted prematurely, leading to partial log collection. This is now fixed by adding the correct wait in the must-gather logic. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1906355[*BZ#1906355*])
 
 * Previously, must-gather collected an unbounded amount of kubelet logs on all nodes. This issue resulted in an excessive amount of data being transferred and collected, with no clear benefit for the user.
 +
 This issue is fixed by collecting a bounded amount, the last eight hours, of kubelet logs only on worker nodes and not collecting kubelet logs on the control plane nodes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1918691[*BZ#1918691*])
 
+* Previously, when the machine config pool was degraded, the performance profile was not updated to display an accurate machine config pool state. Now, the performance profile node selector or machine config pool selector correctly watches the relevant machine config pools, and a degraded machine config pool reflects the correct status. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1903820[*BZ#1903820*])
+
+*RHCOS*
+
+* Previously, configuring additional Azure disks during RHCOS installation caused failure because the `udev` rules for Azure disks were missing from the RHCOS initramfs. The necessary `udev` rules have been added so that configuring additional disks during installation now works properly. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1756173[*BZ#1756173*])
+
+* The `rhcos-growpart.service` has been removed in favor of configuring disks via Ignition at installation time. To change disk configuration after initial RHCOS installation, you must reprovision your systems with the necessary disk configuration changes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1851103[*BZ#1851103*])
```

Comment:
No super actionable feedback other than this one doesn't seem to follow the same format as all the others.

Reply:
I've tweaked this to include the previous situation regarding `rhcos-growpart.service` based on the BZ's description. :+1: